tdrs-backend/gunicorn_start.sh

```diff
@@ -7,4 +7,13 @@ python manage.py migrate
 python manage.py populate_stts
 python manage.py collectstatic --noinput
 echo "Starting Gunicorn"
-exec gunicorn tdpservice.wsgi:application --bind 0.0.0.0:8080 --timeout 10 --workers 3 --log-file=- --log-level $LOGGING_LEVEL
+
+if [[ "$DJANGO_CONFIGURATION" = "Development" ]]; then
```

Comment:
If we set `DJANGO_CONFIG=Development` Like we'd need to in order to reach this and get reloading, we get an error, because our development environment is called "Local"
```suggestion
if [[ "$DJANGO_CONFIGURATION" = "Local" ]]; then
```

Reply:
Our "tanf-dev" environments have this variable set to "Development", so I can't put only "Local" in place. I think it's a reasonable topic for discussion at tabletop. The three possibilities for this variable are prod, dev, local.

Reply:
I figured out what the problem was, there are two ENV vars which we need to tell it which settings to use. `DJANGO_CONFIGURATION` Is one of them, the other is https://docs.djangoproject.com/en/4.0/topics/settings/#envvar-DJANGO_SETTINGS_MODULE

So this won't let us reload locally, cause locally this value will be "LOCAL" because `DJANGO_SETTINGS_MODULE` locally is `      - DJANGO_SETTINGS_MODULE=${DJANGO_SETTINGS_MODULE:-tdpservice.settings.local}
`